policy/MegacamMapper.paf

```diff
@@ -342,37 +294,24 @@ datasets: {
         tables:        raw
     }
 
-    deepCoadd_depth: {
-        template:      "deepCoadd/%(tract)d/%(patch)s.depth.fits"
-        python:        "lsst.afw.image.ImageF"
-        persistable:   "ImageF"
-        storage:       "FitsStorage"
-        tables:        "raw"
-    }
     deep_makeSkyMap_config: {
         template:      "config/deep_makeSkyMap.py"
         python:        "lsst.pipe.tasks.makeSkyMap.MakeSkyMapTask.ConfigClass"
         persistable:   "Config"
         storage:       "ConfigStorage"
         tables:        raw
     }
+    # dataset defined in obs_base modified in this mapper as follows:
```

Comment:
I have this modification on the individual fixes to be done for obs_cfht.  I will do those next.